src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
the rightmost star doesn't need a `margin` 😉

<img width="313" alt="image" src="https://user-images.githubusercontent.com/55802630/172814187-e17dd3b5-131d-4222-a2c0-7e4e8a930bea.png">